packages/jira-adapter/test/change_validators/board_column_config.test.ts

```diff
@@ -0,0 +1,98 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import { toChange, ObjectType, ElemID, InstanceElement } from '@salto-io/adapter-api'
+import { boardColumnConfigValidator } from '../../src/change_validators/board_culomn_config'
+import { BOARD_TYPE_NAME, JIRA } from '../../src/constants'
+
+describe('boardColumnConfigValidator', () => {
+  let type: ObjectType
+  let instance: InstanceElement
+
+  beforeEach(() => {
+    type = new ObjectType({ elemID: new ElemID(JIRA, BOARD_TYPE_NAME) })
```

Comment:
```suggestion
type = createEmptyType(BOARD_TYPE_NAME)
```
And you can also add `createEmptyType(BOARD_TYPE_NAME)` to the instance element constructor without assign it to a variable